Dockerfile

```diff
@@ -44,6 +44,9 @@ ENV LC_ALL="en_US.UTF-8" \
 	GHC_SMTP_SSL=False \
 	GHC_SMTP_USERNAME=None \
 	GHC_SMTP_PASSWORD=None \
+    GHC_GEOIP_URL='http://ip-api.com/json/{hostname}' \
```

Comment:
Can we align/indent accordingly?

Reply:
Aaarrgh. tab vs space discussion.
Fixed with 39183ac